spec/models/dish_spec.rb

```diff
@@ -11,4 +11,41 @@
       it {should belong_to :chef}
   end
 
+  describe "instance methods" do
```

Comment:
Make sure to test all models and their relationships! I don't see a dish_ingredient_spec or an association test added to ingredient_spec.rb. 

Nice work testing the instance methods!